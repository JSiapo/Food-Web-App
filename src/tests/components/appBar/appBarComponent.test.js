import React from 'react';
import AppBarContainer from 'components/appBar';
import renderer from 'react-test-renderer';

it('should compare structure message from AppBar', () => {
  const appBar = renderer.create(<AppBarContainer username={''} />);
  expect(appBar.toJSON).toMatchSnapshot();
});
