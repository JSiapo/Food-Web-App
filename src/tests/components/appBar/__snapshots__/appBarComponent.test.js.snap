// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should compare structure message from AppBar 1`] = `[Function]`;
